src/components/messenger/list/styles.scss

```diff
@@ -45,13 +45,13 @@ $side-padding: 16px;
     display: flex;
     align-items: center;
     justify-content: center;
-    padding-right: 8px;
     width: 100%;
     height: 64px;
     background-color: theme.$color-primary-3;
 
     &--center {
       width: unset;
+      padding-right: 8px;
```

Comment:
This fixes an existing, slightly off-centre rewards button when not in full messenger view.